config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml

```diff
@@ -43,6 +43,35 @@ periodics:
     testgrid-tab-name: unit tests
     testgrid-alert-email: kubernetes-sig-cluster-lifecycle-cluster-api-alerts@googlegroups.com
     testgrid-num-failures-to-alert: "2"
+- name: ci-cluster-api-e2e-test
+  interval: 1h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+    - org: chuckha
+      repo: cluster-api
+      base_ref: e2e-script
+      path_alias: sigs.k8s.io/cluster-api
```

Comment:
Shouldn't this reference `kubernetes-sigs` and `master` rather than `chuckha` and `e2e-script`?

Reply:
lol yes, thanks for catching that haha